### DISCUSSION TITLE
nn.Dropout vs nn.Dropout2d

### DISCUSSION THREAD

**Discussion by Dongbin-Lee-git** [bcaitech1/p4-fr-soccer#53] [tip]
#### nn.Dropout vs nn.Dropout2d
- Encoder FeedForward를 CNN으로 바꾸셨는데 이 부분에서 바꾸어 줄수 있을 것 같네요.
- pretrained cnn이후에 dropout넣어줄 때에 사용할 수 있을 것 같네요.

https://www.programmersought.com/article/8637821427/
